src/services/services.ts

```diff
@@ -4913,6 +4913,24 @@ namespace ts {
 
             if (!documentation) {
                 documentation = symbol.getDocumentationComment();
+                if ((!documentation || documentation.length === 0) && symbol.flags & SymbolFlags.Property) {

```

Comment:
Checked again it actually already enforced always returning an array, so it should be safe to change